### PR TITLE
Seeking paused groups into range now recreates children

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: src/animation-constructor.js [hunk 1]
```diff
@@ -113,6 +113,10 @@
       }
       child.player = this.source.player;
 
+      if (this.paused) {
+          childPlayer.pause();
```
Comment (dstockwell) [inline src/animation-constructor.js]: -2 spaces

File: src/group-constructors.js [hunk 2]
```diff
@@ -55,6 +55,7 @@
     var underlyingPlayer;
     var ticker = function(tf) {
       var player = underlyingPlayer._wrapper;
+      console.log(tf, player.playState, player.startTime);
```
Comment (dstockwell) [inline src/group-constructors.js]: oops

**Comment by dstockwell** [comment]
lgtm otherwise

**Comment by rjwright** [comment]
lgtm after Doug's comments

File: src/animation-constructor.js (hunk 1 above)
Comment (shans) [inline src/animation-constructor.js]: done

File: src/group-constructors.js (hunk 2 above)
Comment (shans) [inline src/group-constructors.js]: done

**Pull request merged**